### ISSUE TITLE
Rename contains to includes?

### ISSUE THREAD

**Issue by paldepind** [ramda/ramda#2447]
Since JavaScript has both [Array#includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) and [String#includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) I was wondering if we should rename `R.contains` to `R.includes`?

Lodash renamed their `_.contains` to `_.includes` a while ago.

It seems that going forward "includes" is going to be what most JavaScript programmers expect this function to be called.

I can do a PR with the renaming and depreciation in case we decide on the renaming.

**Comment by davidchambers** [comment]
Sounds good to me. If one day Ramda provides a generalized function it could be named `elem`, to match Sanctuary (which copied Haskell).

**Comment by CrossEye** [comment]
@paldepind:  I have no major objection, although the story of `contains ~~> includes` because of MooTools' prior art is not a pleasant memory.

@davidchambers:  Do you see `elem` to be more accurate in the general case than `includes` or `contains`?  The name doesn't evoke anything for me, so I don't see why we would choose it.

**Comment by paldepind** [comment]
@CrossEye 

> I have no major objection, although the story of `contains ~~> includes` because of MooTools' prior art is not a pleasant memory.

Interesting. I had no idea that "includes" was chosen due to MooTools. That's definitely not an unfortunate chain of events.

Also, I agree with regards to `elem`. That name isn't particularly intuitive to me either.

**Comment by davidchambers** [comment]
Having `R.contains` as the polymorphic function and `R.includes` as the array- and string- specific function would be confusing. The fact that _elem_ is not synonymous with _includes_ is an advantage, in other words.

**Comment by CrossEye** [comment]
I was assuming that except during a deprecation period, we'd have only one, `includes`.

**Comment by davidchambers** [comment]
Ah. I was proposing one day having both `R.includes` and `R.elem`.

**Comment by semmel** [comment]
I had really hoped to be Ramda's version `0.25` => v. `1.0` (api stable).
Now because `contains` will be probably be purged from a future `1.0` I'll have to convert all `R.contains` in our code base. So sad - I'd really do something else in my time.

I do not doubt the usefulness of this small change, but as a Ramda user who relies on any Ramda functions (e.g. `R.contains`) and seldom uses standard JavaScript functions (e.g. `Array.prototype.includes`) I get bitten by it anyway. ~~So what is the advantage of Ramda then?~~ (just kidding)

tldr; Please release a `1.0`! 🙏

**Comment by paldepind** [comment]
> So sad - I'd really do something else in my time.

Definitely 😭 . For most people, this will hopefully be as simple as a project-wide search and replace from "`R.contains`" to "`R.includes`". And the deprecated `R.contains` will probably stick around for a while so you're in no hurry.

> I do not doubt the usefulness of this small change

That's good to hear 😄
